src/components/ContentSidebar/ActivityFeed/comment/Comment.js

```diff
@@ -156,17 +159,12 @@ class Comment extends React.Component<Props, State> {
                                     <ReadableTime timestamp={createdAtTimestamp} relativeThreshold={ONE_HOUR_MS} />
                                 </small>
                             </Tooltip>
-                            {onEdit && canEditTasksOrComments ? <InlineEdit id={id} toEdit={toEdit} /> : null}
-                            {onDelete && canDeleteTasksOrComments ? (
+                            {onEdit && canEdit ? <InlineEdit id={id} toEdit={toEdit} /> : null}
```

Comment:
Don't worry about actually editing comments, its not supported in the current sidebar so we shouldn't be rendering inline edit at all

Reply:
This is required for Tasks, though. The task component contains a Comment to render text.